maindialog.py

```diff
@@ -153,13 +157,11 @@ def toggleOptions(self):
                         treeOption.setDisabled(False)
 
     def previewMap(self):
-        if not webkit_available:
```

Comment:
That's what I'm running under :) That check has been moved to loadPreviewFile to help with the test suite - so basically now the behavior for with/without webkit is the same right up until the actual preview is shown.